bentoml/tensorflow.py

```diff
@@ -387,19 +389,20 @@ def __init__(
         tag: t.Union[str, Tag],
         predict_fn_name: str,
         device_id: str,
+        partial_kwargs: t.Optional[t.Dict[str, t.Any]],
         resource_quota: t.Optional[t.Dict[str, t.Any]],
         batch_options: t.Optional[t.Dict[str, t.Any]],
         model_store: "ModelStore" = Provide[BentoMLContainer.model_store],
     ):
         super().__init__(str(tag), resource_quota, batch_options)
+        self._device_id = device_id
         self._configure(device_id)
         self._predict_fn_name = predict_fn_name
         assert any(device_id in d.name for d in device_lib.list_local_devices())
+        self._partial_kwargs = partial_kwargs or dict()
```

Comment:
Hmm maybe we shouldn't rely on `or` behaviour. Lets do
```suggestion
        self._partial_kwargs = partial_kwargs if partial_kwargs is not None or dict()
```